run_tests.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Bash sanity settings (error on exit, complain for undefined vars, error when pipe fails)
+set -euo pipefail
+
+CMDNAME="$(basename -- "$0")"
+
+# Update short and long options in the breeze-complete script
+# This way autocomplete will work automagically with all options
+. ${MY_DIR}/run-tests-complete
+
+AIRFLOW_ROOT="$(cd ${MY_DIR}; pwd)"
```

Comment:
```
AIRFLOW_ROOT="$(cd "${MY_DIR}"; pwd)"
```
Double quote to prevent globbing and word splitting.

Reply:
Yep

Reply:
Still outstanding?

Reply:
Isn't this also just the same as `AIRFLOW_ROOT="${MY_DIR}"`?